src/main/python/README.md

```diff
@@ -132,11 +142,11 @@ def sl_pre_load(
 
 ##### StarlakePreLoadStrategy.NONE
 
-No pre load strategy.
+No pre-load strategy: All loads task group will be directly executed. no pre-load tasks group will be created. include reference to [src/main/python/ai/starlake/job/airflow/airflow_starlake_job.py](https://github.com/zedach/starlake/blob/eb67c427a32222d0b83016f01b18862f4ca49ea4/src/main/python/ai/starlake/job/airflow/airflow_starlake_job.py#L134) TODO include Image. 
```

Comment:
It is not totally exact... The pre_tasks method that is called within the pre_load may return a task (at least for Dataproc, there will be a task devoted to the creation of the cluster)

Reply:
I see. In any case, it was an example I put up to show that we need to talk about it, to put an image so that the user knows what to get in the case of NONE. Typically, it's interesting to know that `pre_load` can return a task in the case of the creation of the spark cluster.